easybuild/easyconfigs/o/OpenFOAM/OpenFOAM-v2112-foss-2020b.eb

```diff
@@ -0,0 +1,47 @@
+##
+# Author:    Robert Mijakovic <robert.mijakovic@lxp.lu>
+##
+# Modified by: Ben Langenberg <ben.langenberg@ufz.de>
+##
+name = 'OpenFOAM'
+version = 'v2112'
+
+homepage = 'https://www.openfoam.com/'
+description = """OpenFOAM is a free, open source CFD software package.
+ OpenFOAM has an extensive range of features to solve anything from complex fluid flows
+ involving chemical reactions, turbulence and heat transfer,
+ to solid dynamics and electromagnetics."""
+
+toolchain = {'name': 'foss', 'version': '2020b'}
+toolchainopts = {'cstd': 'c++14', 'vectorize': False}
```

Comment:
```suggestion
# Users have found that vectorizion caused OpenFOAM to produce some very incorrect results.
# Disabling vectorize was confirmed to fix the the known issues.
# With no test suite, sticking to known working toolchain options until proven otherwise.
toolchainopts = {'cstd': 'c++14', 'vectorize': False}